src/components/Semester/Semester.vue

```diff
@@ -408,6 +414,35 @@ export default defineComponent({
     closeConfirmationModal() {
       this.isConfirmationOpen = false;
     },
+
+    saveCourse(course: FirestoreSemesterCourse, collections: string[]) {
+      addCourseToCollections(store.state.currentPlan, this.year, this.season, course, collections);
+      this.openConfirmationModal(`Saved ${course.code} to ${collections.join(', ')}`);
+    },
+    addCollection(name: string) {
+      addCollection(name, []);
+      this.confirmationText = `${name} has been added!`;
+      this.isConfirmationOpen = true;
+      setTimeout(() => {
+        this.isConfirmationOpen = false;
+      }, 2000);
+    },
+    editCollection(oldname: string, name: string) {
+      const { collections } = store.state;
+      const toEdit = collections.find(collection => collection.name === oldname);
+      const updater = (collection: Collection): Collection => ({
+        name,
+        courses: collection.courses,
+      });
+      if (toEdit !== undefined) {
+        editCollection(oldname, updater);
+      }
+      this.confirmationText = `${oldname} has been renamed to ${name}!`;
```

Comment:
Maybe we could force capitalization of the first letter here for UI purposes.

Reply:
For 'oldname'? Maybe we can ask design, but I'd think that we would have that be the exact same as the original name